app/server/lib/DocApi.ts

```diff
@@ -1352,30 +1352,38 @@ export class DocWorkerApi {
     }));
 
     // Do an import targeted at a specific workspace. Although the URL fits ApiServer, this
-    // endpoint is handled only by DocWorker, so is handled here. (Note: this does not handle
-    // actual file uploads, so no worries here about large request bodies.)
+    // endpoint is handled only by DocWorker, so is handled here.
+    // This endpoint either uploads a new file to import, or accepts an existing uploadId.
```

Comment:
Now that actual data is being handled by `handleOptionalUpload`, what worries are there about large request bodies?

Reply:
Functionally no additional ones. This uses an identical mechanism to the `/docs` API, including all the appropriate file-size limits (as configured for the Grist instance).

So we're not introducing any additional issues here, just making the existing functionality accessible via a cleaner endpoint.

Reply:
Works for me!